### ISSUE TITLE
fetchParams.controller.controller.error(new TypeError("terminated", {

### ISSUE THREAD

**Issue by wencan** [MunMunMiao/BilibiliVideoDownload#8]
px bilibili-cli -b BV1NX4y1b7ym -t "..."

os: Debian GNU/Linux 11 (bullseye)  5.10.0-23-amd64 #1 SMP Debian 5.10.179-1 (2023-05-12) x86_64 GNU/Linux

```
npm verb cli /usr/bin/node /usr/lib/node_modules/npm/bin/npm-cli.js
npm info using npm@9.5.1
npm info using node@v18.16.0
npm verb title npm exec bilibili-cli -b BV1NX4y1b7ym -t buvid3=E38E6B9E-781D-4E58-A78E-CA50CF2AE9B5167644infoc; _uuid=109A10EE87-9349-A4B10-9910A-E8E6B72109E10174061infoc; b_nut=100; nostalgia_conf=-1; buvid_fp_plain=undefined; i-4
npm verb argv "exec" "--" "bilibili-cli" "-b" "BV1NX4y1b7ym" "-t" "buvid3=E38E6B9E-781D-4E58-A78E-CA50CF2AE9B5167644infoc; _uuid=109A10EE87-9349-A4B10-9910A-E8E6B72109E10174061infoc; b_nut=100; nostalgia_conf=-1; buvid_fp_plain=un"
npm verb logfile logs-max:10 dir:/home/wencan/.npm/_logs/2023-07-17T12_15_20_602Z-
npm verb logfile /home/wencan/.npm/_logs/2023-07-17T12_15_20_602Z-debug-0.log
npm http fetch GET 200 https://registry.npmjs.org/bilibili-cli 1304ms (cache revalidated)
┌─────────┬───────────┐
│ (index) │  Values   │
├─────────┼───────────┤
│ isLogin │   false   │
│   mid   │ undefined │
│  name   │ undefined │
└─────────┴───────────┘
[Download] [VIDEO] [Title:【古相思曲】01 开播即结局？男女主见面就是“杀招”] [Part:【剧情/爱情】古相思曲（2023）01【张雅钦/郭迦南】] | ■                                        | 3%
node:internal/deps/undici/undici:1132301 开播即结局？男女主见面就是“杀招”] [Part:【剧情/爱情】古相思曲（2023）01【张雅钦/郭迦南】] | ■■■■■■■■■■■■■■■■■■■■■■■                  | 57%
            fetchParams.controller.controller.error(new TypeError("terminated", {
                                                    ^

TypeError: terminated
    at Fetch.onAborted (node:internal/deps/undici/undici:11323:53)
    at Fetch.emit (node:events:513:28)
    at Fetch.terminate (node:internal/deps/undici/undici:10578:14)
    at Object.onError (node:internal/deps/undici/undici:11418:36)
    at Request.onError (node:internal/deps/undici/undici:8210:31)
    at errorRequest (node:internal/deps/undici/undici:10264:17)
    at TLSSocket.onSocketClose (node:internal/deps/undici/undici:9710:9)
    at TLSSocket.emit (node:events:525:35)
    at node:net:322:12
    at TCP.done (node:_tls_wrap:588:7) {
  [cause]: SocketError: other side closed
      at TLSSocket.onSocketEnd (node:internal/deps/undici/undici:9689:26)
      at TLSSocket.emit (node:events:525:35)
      at endReadableNT (node:internal/streams/readable:1359:12)
      at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
    code: 'UND_ERR_SOCKET',
    socket: {
      localAddress: '192.168.1.200',
      localPort: 42504,
      remoteAddress: '120.84.121.84',
      remotePort: 8082,
      remoteFamily: 'IPv4',
      timeout: undefined,
      bytesWritten: 1975,
      bytesRead: 65601760
    }
  }
}

Node.js v18.16.0
npm verb exit 1
npm verb code 1

```

**Comment by MunMunMiao** [comment]
```shell
munmunmiao@MunMuns-MacBook-Pro-4 Downloads % pnpx bilibili-cli -b BV1NX4y1b7ym
Library/pnpm/store/v3/tmp/dlx-67839      |  +31 +++
Library/pnpm/store/v3/tmp/dlx-67839      | Progress: resolved 31, reused 31, downloaded 0, added 31, done
┌─────────┬───────────┐
│ (index) │  Values   │
├─────────┼───────────┤
│ isLogin │   false   │
│   mid   │ undefined │
│  name   │ undefined │
└─────────┴───────────┘
[Download] [VIDEO] [Title:【古相思曲】01 开播即结局？男女主见面就是“杀招”] [Part:【剧情/爱情】古相思曲（2023）01【张雅钦/郭迦南】] | ■■■■■■
[Download] [AUDIO] [Title:【古相思曲】01 开播即结局？男女主见面就是“杀招”] [Part:【剧情/爱情】古相思曲（2023）01【张雅钦/郭迦南】] | ■■■■■■
[Convert] [Title:【古相思曲】01 开播即结局？男女主见面就是“杀招”] [Part:【剧情/爱情】古相思曲（2023）01【张雅钦/郭迦南】] | 1/1
munmunmiao@MunMuns-MacBook-Pro-4 Downloads % 
```

-t option value is from cookie's SESSDATA. not all cookies

**Comment by MunMunMiao** [comment]
i published v0.0.5. you can try again
